internal/gatewayapi/securitypolicy.go

```diff
@@ -851,7 +854,7 @@ func (t *Translator) buildExtAuth(policy *egv1a1.SecurityPolicy, resources *Reso
 	authority = backendRefAuthority(resources, backendRef, policy)
 	pnn := utils.NamespacedName(policy)
 	if ds, err = t.processExtServiceDestination(
-		backendRef,
+		&egv1a1.BackendRef{BackendObjectReference: *backendRef, Failover: priority},
```

Comment:
can we modify this to `Failover: ptr.Deref(grpc.BackendRefs[0].Failover , false)`

Reply:
oh wait, it looks like this line is typecasting from bool from int32 ?

Reply:
We don't have to type cast (its bool type), priority should be named failover.